tests/cxx/typedef_in_template.cc

```diff
@@ -167,6 +173,19 @@ void ArgumentTypeProvision() {
   Outer<Providing>::Inner<IndirectClass*>::AliasedTpl p_ptr;
   // IWYU: IndirectClass needs a declaration
   Outer<IndirectClass*>::Inner<Providing>::AliasedTpl ptr_p;
+
+  // IWYU: IndirectClass needs a declaration
+  // IWYU: IndirectClass is...*indirect.h
+  (void)sizeof(Identity<IndirectClass>::SugaredType);
```

Comment:
This line should probably be moved out of this function, because the function tests slightly different kind of the argument type provision.

Reply:
Yeah, that makes sense to me too.